cmd/kubebuilder/initproject/vendor_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
It's not a big deal, but I have the same question for this line. Do you need to change `2017` to `2018`?

Reply:
don't know :)

Reply:
Would be a good idea.